state_machine:
  process_freq: 20.0

  # takeoff
  takeoff_alt_m: 3.0
  reached_dist_th_m: 0.7 # checking have reached to mission destination waypoint
  guidance_dist_th_m: 4.0 # checking have reached to next waypoint during flight mode

  # landing
  landing_threshold_m: 0.2 # for distinguish kuam control and px4 control during landing 
  landing_standby_alt_m: 3.0 # waiting for detecting marker during landing state
  standby_dist_th_m: 1.5 # checking have reached to landing standby waypoint during landing state
  maf_buf_size: 20 # Moving average filter apply to average pose of 
  big_target_mapping: 0.25
  medium_target_mapping: 0.2
  small_target_mapping: 0.1257

  # flight
  alt_offset_m: 23.156

  # emergy
  battery_th_per: 0.10

state_machine_tf:
  process_freq: 50.0

state_machine_visual:
  process_freq: 5.0

  width: 250.0
  height: 250.0
  left: 10.0
  top: 10.0